models/mongo.js

```diff
@@ -259,6 +259,15 @@ function updateModuleVid(courseID, moduleVid, moduleID, videoID, callback) {
     .catch((err) => callback(err));
 }
 
+function addModuleVid(courseID, moduleVid, moduleID, callback) {
```

Comment:
Haven't stated to you explicitly but we should switch to using async/await to avoid callbacks/modernize the code. There's some examples in other mongo functions

Reply:
This applies to the other ones too